src/observable/from-array.js

```diff
@@ -0,0 +1,59 @@
+var Observable = require("src/Observable");
+
+var FromArrayObservable = function() {
+    
+    function FromArrayObservable(array, scheduler) {
+        this.array = array;
+        this.scheduler = scheduler;
+    }
+    
+    FromArrayObservable.prototype = Object.create(Observable.prototype);
+    
+    FromArrayObservable.prototype._subscribe = _subscribe;
+    
+    function _subscribe(subscriber, state) {
+        
+        var scheduler = state && subscriber || undefined,
+            i = state ? state[1] : -1,
+            array = state ? state[2] : this.array,
+            n = array.length,
+            result = state ? state[3] : { done: false, value: undefined };
+        
+        subscriber = state ? state[0] : subscriber;
+        
+        if(subscriber.disposed) {
+            result.done = true;
+            return result;
+        }
+        
+        if(scheduler) {
```

Comment:
I'm trying to remember... I think the idea was there there would also be a global scheduler. I'm not really sold on that idea, though. But if there were a global scheduling override, we'd need to check it here.